.travis.yml

```diff
@@ -7,3 +7,8 @@ cache: cargo
 matrix:
   allow_failures:
     - rust: nightly
+
+script:
+  - cargo build --verbose
+  - cargo test --verbose
+  - cargo build --verbose --no-default-features
```

Comment:
This is also missing
```yaml
  - cargo test --verbose --no-default-features
```
Maybe I missed it, but aren't you missing `cfg(serialization)` attributes on the serialization test cases too?

Also, what is the advantage of running `cargo build` and `cargo test` as separate tests? I suppose in case the code has `cfg(test)` parts in it?

Reply:
build + test being separate doesn't really have an advantage.  As far as testing with `--no-default-features`, I didn't think it was worthwhile to run the test suite again, and it requires adding another `cfg`, but that's easy enough so I'll go ahead and make that change.

Reply:
Updated.